README.adoc

```diff
@@ -108,7 +110,22 @@ Authenticate 100 JMAP users, one time. Then they all retrieve their system mailb
  - 20% mark mails as ANSWERED
  - 10% mark mails as FLAGGED
 
-=== SmtpNoAuthenticationNoEncryptionMixedBodySimulation
+=== JMAP RFC-8621
+
+==== MailboxGetSimulation
+
+FQDN :
+
+.scala
+----
+org.apache.james.gatling.simulation.jmap.draft.JmapMessageUpdateSimulation
```

Comment:
Wrong FQDN for being in MailboxGetSimulation section I bet

Reply:
Too quick of a bad copy/paste... Thanks!